base/strings/io.jl

```diff
@@ -354,10 +354,11 @@ end
 # TODO: handle unescaping invalid UTF-8 sequences
 """
     unescape_string(str::AbstractString)::AbstractString
-    unescape_string(io, str::AbstractString)::Nothing
+    unescape_string(io, str::AbstractString, keep::AbstractArray{<:AbstractChar} = nothing)::Nothing
```

Comment:
The signature should match the actual one. So either `keep = nothing` (and the text should say "a collection of characters"), or use `::Union{AbstractArray{<:AbstractChar} , Nothing}` in both places.

BTW, below, not only the characters are kept as they are: the whole sequence including the backlash is (also  "as it is" should be "as they are").